states:
  enable_commonstate_batterystatechanged: true
  enable_commonstate_wifisignalchanged: true
  enable_overheatstate_overheatchanged: true
  enable_controllerstate_ispilotingchanged: true
  enable_mavlinkstate_mavlinkfileplayingstatechanged: true
  enable_mavlinkstate_mavlinkplayerrorstatechanged: true
  enable_flightplanstate_availabilitystatechanged: true
  enable_flightplanstate_componentstatelistchanged: true
  enable_pilotingstate_altitudechanged: true
  enable_pilotingstate_flattrimchanged: true
  enable_pilotingstate_flyingstatechanged: true
  enable_pilotingstate_navigatehomestatechanged: true
  enable_pilotingstate_positionchanged: true
  enable_pilotingstate_speedchanged: true
  enable_pilotingstate_attitudechanged: true
  enable_pilotingstate_positionchanged: true
  enable_altitudechanged: true
  enable_autotakeoffmodechanged: true
  enable_mediastreamingstate_videoenablechanged: true
  enable_camerastate_orientation: true
  enable_camerastate_orientationv2: true
  enable_gpsstate_numberofsatellitechanged: true
  enable_numberofsatellitechanged: true

reset_settings: true
publish_odom_tf: true
odom_frame_id: "odom"
cmd_vel_timeout: 0.2
